xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -197,7 +202,7 @@ class GemmAutotuner {
     };
 
     return GetBestAlgorithm<BlasLt::MatmulAlgorithm>(
-        gemm, algorithms, gemm_config.beta, tuned_func);
+        gemm, algorithms, gemm_config.beta, true, tuned_func);
```

Comment:
I would really prefer to remove this parameter (see other comment). If it truly has to stay, then please add `/*return_algo_index=*/` in front.